nixos/doc/manual/release-notes/rl-2009.xml

```diff
@@ -72,6 +72,38 @@
 }</programlisting>
     </para>
    </listitem>
+
+   <listitem>
```

Comment:
Maybe worth adding a section on how to use the new option?  I see from below that it's `/run/current-system/specialisation/NAME/bin/switch-to-configuration` instead of `fine-tune/child-#`